install/helm/agones/templates/controller.yaml

```diff
@@ -13,6 +13,13 @@
 # limitations under the License.
 
 {{- $featureGates := include "agones.featureGates" . | fromYaml }}
+{{- $replicas := (int .Values.agones.controller.replicas) }}
+{{- if not $featureGates.SplitControllerAndExtensions }}
+{{- $replicas = 1 }}
+{{- end }}
+{{- if eq $replicas 0 }}
```

Comment:
paranoid nit: might as well check replicas < 1 in case someone tries something silly like a negative

Reply:
Done